specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2020-03-01/managedClusters.json

```diff
@@ -1426,10 +1429,7 @@
         "count": {
           "type": "integer",
           "format": "int32",
-          "maximum": 100,
-          "minimum": 0,
-          "description": "Number of agents (VMs) to host docker containers. Allowed values must be in the range of 0 to 100 (inclusive). The default value is 1. ",
-          "default": 1
```

Comment:
Removing default causes breaking change. Can we keep this?

Reply:
@NullMDR per the PR description, those defaults shouldn't have been there in the first place. 

https://github.com/Azure/azure-rest-api-specs/pull/7746 removed those defaults but when we branched off for November API, we didn't include those PR.

Reply:
The changes here are just a cherry-pick of #7746 which is what the state of the spec should be.